checklist.html

```diff
@@ -1,242 +1,1029 @@
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html">1.4.1 Use of Color</a>
+        </p>
+        <p class="checklist-description">
+          People who are color blind, who have other low vision conditions, or different cultural understandings for color may not see the state change, or understand what kind of feedback the state represents if color is the only indicator.
+        </p>
+      </div>
+    </details>
+    <details id="section-forms-07">
+      <summary>
+        Update the page or view's <code>title</code> to include high-level form feedback.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          The <code>title</code> element's content is often the first content announced by a screen reader. Adding feedback about the presence of things like errors or warnings after form submission allows people to know there are items on the page or view that need their attention.
+        </p>
+      </div>
+    </details>
+  </div>
+</section>
+
+<section class="article-section resources-section checklist-section" id="section-media" aria-labelledby="label-media">
+  <h2 id="label-media">Media</h2>
+  <p>
+    Media includes content such as pre-recorded and live audio and video.
+  </p>
+  <div class="checklist">
+    <details id="section-media-01">
+      <summary>
+        Make sure that media does not autoplay.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-dis-audio.html">1.4.2 Audio Control</a>
+        </p>
+        <p class="checklist-description">
+          Unexpected video and audio can be distracting and disruptive, especially for certain kinds of cognitive disability such as ADHD. Certain kinds of autoplaying video and animation can be a trigger for vestibular and seizure disorders.
+        </p>
+      </div>
+    </details>
+    <details id="section-media-02">
+      <summary>
+        Ensure that media controls use appropriate markup.
+      </summary>
+      <div class="checklist-details">
+        <p class="checklist-criterion">
+          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html">1.3.1 Info and Relationships</a>
+        </p>
+        <p class="checklist-description">
+          Examples include making sure a video play button has a pressed toggle state when active, or that a volume slider uses <code>&#x3C;input type=&#x22;range&#x22;&#x3E;</code>.
```

Comment:
i'd swap out "play" for "mute" here.

play buttons more typically toggle their accessible name with play/pause, so a "toggle button" e.g. `aria-pressed=t/f` wouldn't be applicable here.

Reply:
Agreed, though Léonie recently described her frustration with Play controls and the accessible name changing; she'd rather have the `aria-pressed` applied. 🤷‍♂️

https://youtu.be/OUDV1gqs9GA?t=3220

Reply:
Right, and from a completely visually impaired perspective that makes sense.  but as the visual icon of a play/pause often changes as well, it'd be just as confusing for other AT users if the accessible name remained play, but the button displayed a pause icon.

hence change the example to mute, since that's no need to change that label and it fits the pressed / not pressed example better.

Reply:
Done.